src/WebUI/Components/ResourceStatusComponent.tsx

```diff
@@ -15,16 +15,33 @@ export interface IResourceStatusProps extends IVssComponentProperties {
 export class ResourceStatusComponent extends BaseComponent<IResourceStatusProps, {}> {
 
     public render(): React.ReactNode {
+        if (this.props.toolTipText) {
```

Comment:
should we do this like


retrurn this.props.toolTipText ? this._getStatusWithToolTip() : this._getStatus();